src/app/dashboard/components/dashboard-new-events/dashboard-new-events.component.html

```diff
@@ -1 +1,43 @@
-<p>dashboard-new-events works!</p>
+<div class="dashboard-new-events" *ngIf="newEvents">
+  <h1>
+    <span class="nr-of-events">{{
+      newEvents.CLUSTER +
+        newEvents.EVENT +
+        newEvents.SIGNAL +
+        newEvents.SCREENING +
+        newEvents.DROPPED || 0
```

Comment:
please create a method here to be more clear

Reply:
Fixed.